NAMESPACE

```diff
@@ -1,14 +1,17 @@
-useDynLib(alphashape3d)
-import(geometry) 
-import(rgl) 
+# Generated by roxygen2: do not edit by hand
+
+S3method(plot,ashape3d)
 export(ashape3d)
 export(components_ashape3d)
 export(inashape3d)
-export(plot.ashape3d)
 export(rtorus)
```

Comment:
One other point. Traditional R style would hold that `plot.ashape3d` should not be exported as a function but only as an S3 `plot` method for class `ashape3d`. Your NAMESPACE file previously did both. When I switched to roxygen the direct export i.e. `export(plot.ashape3d)` was a casualty. You may want to consider if this is acceptable.